client/state/sites/media-storage/schema.js

```diff
@@ -0,0 +1,15 @@
+export const itemsSchema = {
+	type: 'object',
+	patternProperties: {
+		//be careful to escape regexes properly
+		'^\\d+$': {
+			type: 'object',
+			required: [ 'max_storage_bytes', 'storage_used_bytes' ],
+			properties: {
+				max_storage_bytes: { type: [ 'integer', 'number' ] },
```

Comment:
The endpoint may return a float if we increase our plan sizes in the future, but this shouldn't happen. I'll take this out.